rpm/readline.spec

```diff
@@ -1,9 +1,14 @@
+%define rl_major 8
+%define rversion 8.2
+%define rpatchlvl 10
```

Comment:
None of these are needed either, no point adding extra variables just keep on the old way that we have used

Reply:
The variables indicate the patch level of readline. The base version only shows the version without any patches. The upstream does not bump the release for those.
The variables illustrate that.